pkg/apis/pipeline/v1beta1/pipelinerun_validation.go

```diff
@@ -112,6 +112,12 @@ func (ps *PipelineRunSpec) Validate(ctx context.Context) (errs *apis.FieldError)
 		}
 	}
 
+	if ps.TaskRunSpecs != nil {
+		for idx, trs := range ps.TaskRunSpecs {
```

Comment:
the `range` keyword will handle `nil` values without panicking, no need to wrap in an `if` I don't think.

Reply:
good catch thanks!